packages/core/package.json

```diff
@@ -1,8 +1,40 @@
 {
   "name": "@bugsnag/core",
-  "main": "index.js",
+  "main": "src/index.js",
```

Comment:
In this PR the package source is moved to a `src` folder but the package exports are still referring to the source (not the generated TypeScript). This allows the file move to be separated from the TypeScript conversion allowing for smaller PRs that are easier to review